index.html

```diff
@@ -1,28 +1,101 @@
 <!DOCTYPE html>
 <html>
 <head>
-<meta charset=utf-8 />
-<title>DC Bike Infrastructure</title>
-
-<link href='https://www.mapbox.com/base/latest/base.css' rel='stylesheet' />
-<meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no' />
-<script src='https://api.tiles.mapbox.com/mapbox.js/v2.4.0/mapbox.js'></script>
-<link href='https://api.tiles.mapbox.com/mapbox.js/v2.4.0/mapbox.css' rel='stylesheet' />
-<link href='styles.css' rel='stylesheet' />
+    <!-- ======= params ======= -->
+    <meta charset=utf-8 />
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+    <title>DC Bike Infrastructure</title>
+
+    <!-- ======= mapbox ======= -->
+    <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet-0.7.3/leaflet.css" />
+    <link rel="stylesheet" href="https://www.mapbox.com/base/latest/base.css" />
+    <!-- <link rel="stylesheet" href="https://api.tiles.mapbox.com/mapbox.js/v2.4.0/mapbox.css" /> -->
```

Comment:
Why was the Mapbox.js stylesheet removed? I think this is necessary for loading Mapbox.js maps.